app/routes/circulars/circulars.server.ts

```diff
@@ -52,6 +52,40 @@ function parseDate(date?: string) {
   return date ? new Date(date).getTime() : NaN
 }
 
+/** take input string and return start/end times based on string value */
+function fuzzyTimeRange(fuzzyTime?: string) {
+  const now = Date.now()
+  if (fuzzyTime === 'now') return now
+  if (fuzzyTime === 'hour') return now - 3600000
+  if (fuzzyTime === 'today') return new Date().setHours(0, 0, 0, 0)
+  if (fuzzyTime === 'day') return now - 86400000
+  if (fuzzyTime === 'week') return now - 86400000 * 7
+  if (fuzzyTime === 'month') return now - 86400000 * 30
+  if (fuzzyTime === 'year') return now - 86400000 * 365
+  if (fuzzyTime === 'mtd')
+    return new Date(
+      new Date().getFullYear(),
+      new Date().getMonth(),
+      1
+    ).getTime()
+  if (fuzzyTime === 'ytd')
+    return new Date(new Date().getFullYear(), 0).getTime()
+  else return undefined // invalid fuzzyTime defaults to fuzzless time range
```

Comment:
Not necessary. If the function never executes a `return` statement, then the return value is undefined.

```suggestion
```